apis/mattermost/v1beta1/mattermost_utils.go

```diff
@@ -113,6 +124,18 @@ func (mm *Mattermost) GetIngressHostNames() []string {
 	return hosts
 }
 
+func (mm *Mattermost) GetAWSLoadBalancerHostNames() []string {
```

Comment:
I think we do not need this method. The caller can get the the list from spec.

Reply:
@mirshahriar I was following: https://github.com/mattermost/mattermost-operator/blob/63f7b1d59b18835be02b190aaeefba24972f90fc/pkg/mattermost/mattermost_v1beta.go#L136